test/e2e/default_bridged_network_test.go

```diff
@@ -48,7 +48,7 @@ var _ = Describe("NodeNetworkConfigurationPolicy default bridged network", func(
 `)
 	// FIXME: This is a pending spec since we have to discover why the
 	//        cluster never goes back at kubevirtci provider
-	XContext("when there is a default interface with dynamic address", func() {
+	Context("when there is a default interface with dynamic address", func() {
```

Comment:
If the spec is now working the comment should be removed?

Reply:
Nah it's not working :-/

Reply:
@fedepaol now should work let me remove it.